topics/transcriptomics/tutorials/unaligned-ncrnas-analysis/tutorial.md

```diff
@@ -0,0 +1,518 @@
+---
+layout: tutorial_hands_on
+
+title: Analyse unaligned ncRNAs
+zenodo_link: 'https://zenodo.org/record/3482616'
+questions:
+- Which biological questions are addressed by the tutorial?
+  - Is my set of RNAs of interest homologous?
+  - Are the RNAs in the set coding for a protein?
+- Which bioinformatics techniques are important to know for this type of data?
+  - Alignment
+  - RNA secondary structure folding
+objectives:
+- Knowing for which cases the "Analyze non coding RNAs" workflow can be used
+- Being able to apply the analyze non coding RNAs workflow
+- Learn about the tools used in the workflow
+time_estimation: 1H
+key_points:
+- Homology
+- RNA
+- Secondary structure
+- Coding potential
+contributions:
+  authorship:
+    - eggzilla
+    - bebatut
+  editing:
+    - pavanvidem
```

Comment:
(feel free to move yourself to authorship if this turns out to be more work than just a quick test @pavanvidem, just wanted to change this to use the new-style `contributions` metadata)